lib/RefBackend/Runtime/Runtime.cpp

```diff
@@ -293,5 +386,59 @@ LogicalResult refbackrt::getMetadata(ModuleDescriptor *moduleDescriptor,
     return failure();
   outMetadata.numInputs = descriptor->numInputs;
   outMetadata.numOutputs = descriptor->numOutputs;
+
+  for (int i = 0; i < descriptor->numInputs; i++) {
+    outMetadata.inputArgInfos[i] = getExternalInputArgInfo(descriptor->inputDescriptors[i]);
+  }
+
+  for (int i = 0; i < descriptor->numOutputs; i++) {
+    outMetadata.outputArgInfos[i] = getExternalOutputArgInfo(descriptor->outputDescriptors[i]);
+  }
+
   return success();
 }
+
+RtValue refbackrt::createRtValueFromOutputArgInfo(const OutputArgInfo& info) {
```

Comment:
I don't understand why this function is needed. refbackrt::invoke should have already created the rt values.

Reply:
Resolved in comment below (github UI appears to be functioning properly again)